Kotlin. Nested configurations do not work in 0.14.4

Hi there!

Version 0.14.3, works fine. In addition to issue title I have a message when IntelleJ IDE sync Gradle project
```
org.gradle.internal.exceptions.LocationAwareException: Build file '/Users/user_name/Projects/my_project/build.gradle.kts' line: 222
Receiver class com.avast.gradle.dockercompose.ComposeSettings does not define or inherit an implementation of the resolved method 'abstract java.lang.Object getProperty(java.lang.String)' of interface groovy.lang.GroovyObject.
	...
	at Program.execute(Unknown Source)
Caused by: java.lang.AbstractMethodError: Receiver class com.avast.gradle.dockercompose.ComposeSettings does not define or inherit an implementation of the resolved method 'abstract java.lang.Object getProperty(java.lang.String)' of interface groovy.lang.GroovyObject.
	at com.avast.gradle.dockercompose.DockerExecutor.<init>(DockerExecutor.groovy:15)
	...
	at com.avast.gradle.dockercompose.ComposeSettings.<init>(ComposeSettings.groovy:120)
	...
	at com.avast.gradle.dockercompose.ComposeSettings.cloneAsNested(ComposeSettings.groovy:140)
	...
	at com.avast.gradle.dockercompose.ComposeExtension$_getOrCreateNested_closure1.doCall(ComposeExtension.groovy:14)
	...
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1133)
	...
	at com.avast.gradle.dockercompose.ComposeExtension.getOrCreateNested(ComposeExtension.groovy:14)
	at com.avast.gradle.dockercompose.ComposeExtension.nested(ComposeExtension.groovy:22)
	at Build_gradle$8.execute(build.gradle.kts:222)
	at Build_gradle$8.execute(build.gradle.kts:3)
	at org.gradle.internal.extensibility.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:173)
	at org.gradle.internal.extensibility.ExtensionsStorage.configureExtension(ExtensionsStorage.java:64)
	at org.gradle.internal.extensibility.DefaultConvention.configure(DefaultConvention.java:194)
	at org.gradle.kotlin.dsl.Accessors8cjdgd6airkzom6u1qviexhyzKt.dockerCompose(Unknown Source)
	at Build_gradle.<init>(build.gradle.kts:221)
	... 151 more

```

Thanks for the great plugin.

--- a/src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy
+++ b/src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy
@@ -8,6 +8,7 @@ import com.avast.gradle.dockercompose.tasks.ComposePull
 import com.avast.gradle.dockercompose.tasks.ComposePush
 import com.avast.gradle.dockercompose.tasks.ComposeUp
 import com.avast.gradle.dockercompose.tasks.ServiceInfoCache
+import groovy.transform.CompileStatic
 import groovy.transform.PackageScope
 import org.gradle.api.Project
 import org.gradle.api.Task
@@ -21,6 +22,7 @@ import java.nio.charset.StandardCharsets
 import java.security.MessageDigest
 import java.time.Duration
 
+@CompileStatic
 class ComposeSettings {
     final TaskProvider<ComposeUp> upTask
     final TaskProvider<ComposeDown> downTask
@@ -109,13 +111,13 @@ class ComposeSettings {
         this.project = project
         this.nestedName = parentName + name
 
-        upTask = project.tasks.register(name ? "${name}ComposeUp" : 'composeUp', ComposeUp, { it.settings = this })
-        buildTask = project.tasks.register(name ? "${name}ComposeBuild" : 'composeBuild', ComposeBuild, { it.settings = this })
-        pullTask = project.tasks.register(name ? "${name}ComposePull" : 'composePull', ComposePull, { it.settings = this })
-        downTask = project.tasks.register(name ? "${name}ComposeDown" : 'composeDown', ComposeDown, { it.settings = this })
-        downForcedTask = project.tasks.register(name ? "${name}ComposeDownForced" : 'composeDownForced', ComposeDownForced, { it.settings = this })
-        logsTask = project.tasks.register(name ? "${name}ComposeLogs" : 'composeLogs', ComposeLogs, { it.settings = this })
-        pushTask = project.tasks.register(name ? "${name}ComposePush" : 'composePush', ComposePush, { it.settings = this })
+        upTask = project.tasks.register(name ? "${name}ComposeUp".toString() : 'composeUp', ComposeUp, { it.settings = this })
+        buildTask = project.tasks.register(name ? "${name}ComposeBuild".toString() : 'composeBuild', ComposeBuild, { it.settings = this })
+        pullTask = project.tasks.register(name ? "${name}ComposePull".toString() : 'composePull', ComposePull, { it.settings = this })
+        downTask = project.tasks.register(name ? "${name}ComposeDown".toString() : 'composeDown', ComposeDown, { it.settings = this })
+        downForcedTask = project.tasks.register(name ? "${name}ComposeDownForced".toString() : 'composeDownForced', ComposeDownForced, { it.settings = this })
+        logsTask = project.tasks.register(name ? "${name}ComposeLogs".toString() : 'composeLogs', ComposeLogs, { it.settings = this })
+        pushTask = project.tasks.register(name ? "${name}ComposePush".toString() : 'composePush', ComposePush, { it.settings = this })
 
         this.dockerExecutor = new DockerExecutor(this)
         this.composeExecutor = new ComposeExecutor(this)

